Menu/Menu.js

```diff
@@ -1,10 +1,18 @@
-
 const toggleMenu = () => {
-  // Toggle the "menu--open" class on your menu refence. 
+  // Toggle the "menu--open" class on your menu refence.
+  menu.classList.toggle('menu--open') 
+}
+const animateMenuOut = () => {
+  TweenMax.to(menu, 1, {x:350}); 
+}
+const animateMenuIn = () => {
+  TweenMax.to(menu, 1, {x:-350});
```

Comment:
I love the effect you added to the side menu here and how it slides back in when you aren't hovering over it! 🔥